radis/tools/database.py

```diff
@@ -2372,7 +2372,11 @@ def _load_new_file(self, file, binary=False):
 
         out = s.get_conditions().copy()
 
-        # Add filename, and a link to the Spectrum object itself
+        # Add filename, name and a link to the Spectrum object itself
+        if s.name == None:
+            out["name"] = s.get_name()
```

Comment:
@erwanp, I'm actually using this to store the name column in the csv file instead of updating the `conditions`. If `s.name==None` it stores the filename else the name given to it. It gets the job done. Any better location for doing this?

Reply:
Good idea! Let's keep it here!